src/main/resources/static/test-sessions.json

```diff
@@ -0,0 +1 @@
+[{"name":"session_1","step":"MAPPING","savedDate":1433333052299},{"name":"session_2","step":"MAPPING","savedDate":1438257852299},{"name":"session_session_session_session_session_session_session_session_3","step":"MAPPING","savedDate":1439640252299},{"name":"session_4","step":"PATIENTS","savedDate":1442318652299},{"name":"session_5","step":"PATIENTS","savedDate":1444910652299},{"name":"session_6","step":"EVENTS","savedDate":1447592652299},{"name":"session_7","step":"EVENTS","savedDate":1450443852299},{"name":"session_8","step":"OVERVIEW","savedDate":1451653452299},{"name":"session_9","step":"EVENTS","savedDate":1455541452299}]
```

Comment:
This red crossed circle with return is a sign that you forgot to leave empty line at the end of your file. You can configure Intellij to do it automatically.